README.md

```diff
@@ -65,3 +65,12 @@ Importing any stylesheet from JS (with CSS Modules enabled) will convert class n
   @import '~react-select/dist/react-select';
 }
 ```
+#### 5. SVG as JSX, is it even possible?
+Importing SVGs as JSX is possible, but only SVGs suffixed with `.inline` will be treated as JSX code (fe. `logo.inline.svg`):
```

Comment:
Since you're showing an example using `logo.inline.svg` just below, the crossed out part is unnecessary, I think:
> Importing SVGs as JSX is possible, but only SVGs suffixed with `.inline` will be treated as JSX code ~(fe. `logo.inline.svg`)~: